docs/integrations/ai-engines/anyscale.mdx

```diff
@@ -34,13 +34,108 @@ CREATE MODEL anyscale_endpoints_model
 PREDICT target_column
 USING
       engine = 'anyscale_endpoints_engine',   -- engine name as created via CREATE ML_ENGINE
-      model_name = 'model-name',              -- choose one of available models
-      prompt_teplate = 'prompt-to-the-model'; -- prompt message to be completed by the model
+      mode = 'conversational', -- optional, mode to run the model in
+      model_name = 'anyscale_endpoints_model_name',  -- optional, the LLM to use
+      prompt = 'You are a helpful assistant. Your task is to continue the chat.',  -- optional, system prompt for the model
+      question_column = 'question',  -- optional, column name that stores user input
+      context_column = 'context',  -- optional, column that stores context of the user input
+      prompt_template = 'Answer the users input in a helpful way: {{question}}', -- optional, base template with placeholders used to provide input to the model 
+      max_tokens = 100, -- optional, token limit for model output
+      temperature = 0.3, -- optional, randomness setting for the model output
+      json_struct = {
+        'key': 'value',
+        ...
+      }' -- optional, the parameter for extracting JSON data from `prompt_template`
 ```
 
+<Tip>
+It is possilbe to override certain parameters set for a model at prediction time instead of recreating the model. For example, to change the temperature parameter for a specific prediction, use the following query:
```

Comment:
Minor update: `...to change the prompt_template and temperature parameters for...`

Reply:
Yes. Currently we specify what can be overriden at prediction time in the following Accordion section. That's fine.